tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -927,6 +930,27 @@ def setUp(self):
         else:
             self.cleanup_gaia(full_reset=True)
 
+        # if video capture is enabled, then start recording
+        if self.capture:
+            self.start_video_capture()
+
+    # saves the captured video in /sdcard/ folder (only logical choice)
+    # trigger screenrecord command as a background process since this does not return until explicitly killed
+    # (this is why os.system() was used instead of device.manager.shell(),
+    # since shell() cannot start a background process
+    def start_video_capture(self):
+        self.capture_file = '%s_%s.mp4' % (self.methodName, datetime.datetime.now().strftime('%Y-%m-%d-%H-%M-%S'))
+        os.system("adb shell screenrecord " + os.path.join("/sdcard", self.capture_file) + "&")
+
+    # stops the capture and pulls the video file
+    def stop_video_capture(self):
```

Comment:
The comment 1 line above is not necessary if we rename this function to `stop_and_pull_video_capture`